Jenkinsfile

```diff
@@ -124,36 +126,38 @@ node('vetsgov-general-purpose') {
     }
   }
 
-  stage('Lint|Security|Unit') {
-    try {
-      parallel (
-        lint: {
-          dockerImage.inside(args) {
-            sh "cd /application && npm --no-color run lint"
-          }
-        },
-
-        // Check package.json for known vulnerabilities
-        security: {
-          retry(3) {
+  if (cmsEnv == 'none') {
```

Comment:
Not sure why the diff is so crazy here, I just surrounded this stage with an if statement, since we don't need to run the tests if we're using pre-built assets.

Reply:
We'll want to put the if inside the `stage()` here, otherwise Jenkins will probably re-draw all the stages in the UI and make it difficult to see how long each stage takes when we go looking.

Something like:
`if (cmsEnv != 'none') { return }`